deployment/src/main/java/io/quarkus/fury/deployment/FuryProcessor.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.fury.deployment;
```

Comment:
```suggestion
package io.quarkiverse.fury.deployment;
```

Reply:
Rename the other packages too to use `io.quarkiverse.fury`

Reply:
done